packages/summary-list/src/SummaryList.tsx

```diff
@@ -0,0 +1,60 @@
+import { ReactNode } from 'react';
+import { Flex } from '@ag.ds-next/box';
+
+type PropsWithOnlyChildren = {
+	children: ReactNode;
+};
+
+export type SummaryListProps = PropsWithOnlyChildren;
+
+/** SummaryListItemTerm should be nested as the first child of SummaryListItem */
+export const SummaryListItemTerm = ({ children }: PropsWithOnlyChildren) => (
+	<Flex as="dt" fontWeight="bold" width={['100%', '30%']} minWidth="200px">
+		{children}
+	</Flex>
+);
+
+/** SummaryListItemTerm should be nested as the second child of SummaryListItem, after SummaryListItemTerm */
+export const SummaryListItemDescription = ({
+	children,
+}: PropsWithOnlyChildren) => (
+	<Flex as="dd" flexGrow={1}>
+		{children}
+	</Flex>
+);
+
+/** SummaryListItemAction should be nested as the last child of SummaryListItem, and should wrap a Link component */
+export const SummaryListItemAction = ({ children }: PropsWithOnlyChildren) => (
+	<Flex as="dd">{children}</Flex>
+);
+
+/**
+ * SummaryList displays a list of terms and descriptions as key value pairs.
+ * It is similar to a two-column table, but renders a HTML Description List (`<dl>`) element.
+ */
+export function SummaryList({ children }: SummaryListProps) {
```

Comment:
Some of these component definitions are inconsistent. Both styles (`export function Component() {}` and `export const ComponentName = () =>{}`) are being used.